libbeat/docs/monitoring/configuring.asciidoc

```diff
@@ -0,0 +1,178 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[role="xpack"]
+[[configuring-monitoring]]
+== Configuring monitoring for {beatname_uc}
+++++
+<titleabbrev>Configure monitoring</titleabbrev>
+++++
+
+{monitoring} enables you to easily monitor {beatname_uc} from {kib}. For more
+information, see {kibana-ref}/xpack-monitoring.html[Monitoring].
+
+. Configure {beatname_uc} to send metrics to your {es} cluster by setting the
+`xpack.monitoring` settings in the {beatname_uc} configuration file. For
+example:
++
+--
+[source, yml]
+--------------------
+xpack.monitoring:
+  enabled: true
+  elasticsearch:
+    url: ["https://example.com:9200", "https://example2.com:9200"]
+    username: elastic
+    password: changeme
+--------------------
+
+Many of the configuration options for `xpack.monitoring` match the
+`output.elasticsearch` options. If you configured {es} output and you want to
+use the same {es} cluster and credentials to store your monitoring metrics, you
+can specify the following minimal configuration options:
```

Comment:
Note: the monitoring data must be shipped to the production cluster. It is the x-pack production cluster forwarding the metrics to the monitoring cluster. The hosts should only be configured if another output then elasticsearch is configured (e.g. Kafka or Logstash).